File: python/mqtt_test.py
import paho.mqtt.client as mqtt  #import the client1
import time

def on_connect(client, userdata, flags, rc):
    m="Connected flags"+str(flags)+"result code "\
    +str(rc)+"client1_id  "+str(client)
    print(m)

def on_message(client1, userdata, message):
    print("message received  "  ,str(message.payload.decode("utf-8")))

broker_address="192.168.1.102"
#broker_address="iot.eclipse.org"
client1 = mqtt.Client("P1")    #create new instance
client1.on_connect= on_connect        #attach function to callback
client1.on_message=on_message        #attach function to callback
time.sleep(1)
client1.connect(broker_address)      #connect to broker
client1.loop_start()    #start the loop
#client1.subscribe("house/bulbs/bulb1")
client1.publish("hello/world","wft")
time.sleep(5)
client1.disconnect()
client1.loop_stop()

File: node/coord/mqtt_test.js
var mqtt = require('mqtt')
//var client = mqtt.connect()

var client = mqtt.connect({ port: 1883, host: '192.168.1.102'});

//client.on('conn');

//client.subscribe('presence')
client.publish('hello/world', 'bin hier')

client.on('message', function (topic, message) {
  console.log(message)
})

client.end()
